samples/Samples.Schemas.Chat/Samples.Schemas.Chat.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <WarningsNotAsErrors>CS8618</WarningsNotAsErrors>
+    <WarningsNotAsErrors>CS8618;CA1716</WarningsNotAsErrors>
```

Comment:
I had to add new ignore explicitly sine you do not want to be linked to root props files